src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -104,22 +104,38 @@ public static void updateUserInDatastore(String identifyingProperty, String iden
     datastore.put(entity);
   }
 
+  public static GivrUser getUserByIdOrEmail(String userId, String userEmail) {
```

Comment:
Do you intend to have this public or do you want this to be private?

Reply:
I do not see any uses for keeping it public, for now. I've changed this to be a private method.